database/migrations/2025_01_09_194600_create_employees_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('employee', function (Blueprint $table) {
```

Comment:
Are we using singular for database names? 
```suggestion
        Schema::create('employees', function (Blueprint $table) {
```